ecosystem.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  apps: [
+    {
+      name: 'Aptiche Server',
+      script: './dist/index.js',
+      watch: false,
+      instances: 1, // TODO: change to max for production
+      exec_mode: 'cluster',
+      max_memmory_restart: '500M',
+      time: false,
+      error_file: './logs/pm2-err.log',
+      out_file: './logs/pm2-out.log',
+      log_file: './logs/pm2-app.log',
+      combine_logs: false,
+      kill_timeout: 2000,
+      min_uptime: '1m',
+      max_restarts: 5,
+      restart_delay: 1000,
+      autorrestart: true,
+    },
```

Comment:
There is a thing known as an exponential back-off restart. Use that.

I see that is watch is false here, what do you plan for development phase?

Reply:
Planning not to use pm2 during dev but will include this if you suggest.